go.mod

```diff
@@ -9,7 +9,26 @@ require (
 	github.com/spf13/viper v1.8.1
 	github.com/stretchr/testify v1.7.0
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
+)
+
+require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
```

Comment:
@chenrui333 has `go mod tidy` been run on this branch?

Reply:
Yes! this is normal for go1.17 (which separate the direct deps and indirect deps)

Reply:
Sorry for the delay @chenrui333, updated to go1.17 myself and am seeing the same things. Thanks for this PR. Approving.